presto-connector-tests/src/main/java/com/facebook/presto/connector/unittest/BaseMetadataTest.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.connector.unittest;
```

Comment:
I think this sounds like a job for somebody actually taking the time to make the package names consistent when they're adding unittest.planner rather than me trying to guess what that person is thinking in the future ;-)